docs/more/faq.rst

```diff
@@ -180,7 +121,7 @@ For support pricing, please `contact us <https://osohq.com/company/contact-us>`_
 License
 -------
 
-oso licensed under the `Apache 2.0 license <https://github.com/osohq/oso/blob/master/LICENSE>`.
+oso licensed under the _`Apache 2.0 license <https://github.com/osohq/oso/blob/master/LICENSE>`_.
```

Comment:
Were you trying to italicize this? I think that needs to be:

```suggestion
oso licensed under the *`Apache 2.0 license <https://github.com/osohq/oso/blob/master/LICENSE>`_*.
```
That final `_` denotes it as an external link .....